smartling-job-batches-api/src/main/java/com/smartling/api/jobbatches/v2/pto/BatchItemStatus.java

```diff
@@ -0,0 +1,21 @@
+package com.smartling.api.jobbatches.v2.pto;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public enum BatchItemStatus
```

Comment:
Think, it can be simplified as:
```
public enum BatchItemStatus {
    UPLOADING,
    UPLOAD_FAILED,
    ATTACHING,
    ATTACH_FAILED,
    COMPLETED,
    DRAFT,
    CANCELED;
}
```

Reply:
Good catch. Thank you, Pavel!